content/docs/csidriver/features/powerscale.md

```diff
@@ -394,9 +402,9 @@ parameters:
   IsiPath: "/ifs/data/csi"
   # AccessZone groupnet service IP. Update AzServiceIP in values.yaml if different than isiIP.
   #AzServiceIP : 192.168.2.1
-  # When a PVC is being created, it takes the storage class' value of "storageclass.rootClientEnabled", 
-  # which  determines, when a node mounts the PVC, in NodeStageVolume, whether to add the k8s node to 
-  # the "Root clients" field (when true) or "Clients" field (when false) of the NFS export 
+  # When a PVC is being created, it takes the storage class' value of "storageclass.rootClientEnabled",
+  # which  determines, when a node mounts the PVC, in NodeStageVolume, whether to add the k8s node to

```

Comment:
Remove comma after "storageclass.rootClientEnabled". Insert a period. Start a new sentence which reads: "INodeStageVolume determines when a node mounts the PVC,